code/game/machinery/cloning.dm

```diff
@@ -126,9 +126,21 @@
 	if (is_operational() && (!isnull(occupant)) && (occupant.stat != DEAD))
 		to_chat(user, "Current clone cycle is [round(get_completion())]% complete.")
 
+/obj/machinery/clonepod/return_air()
+	// We want to simulate the clone not being in contact with
+	// the atmosphere, so we'll put them in a constant pressure
+	// nitrogen. They'll breathe through the chemicals we pump into them.
+	var/datum/gas_mixture/GM = new

```

Comment:
it's not too hard, I did it once for a scrapped PR.
most of the space-specific things look obviously space-specific, and you just don't transfer those parts across.